resources/lang/en/users.php

```diff
@@ -53,6 +53,7 @@
         '404' => 'User not found! ;_;',
         'age' => ':age years old',
         'current_location' => 'Currently in :location',
+        'dateFormat' => 'MMMM YYYY',
```

Comment:
Use camel case here, also name it more explicitly, e.g. `'joined_at_format' => ...`